lms/static/js/utils/dropdown.js

```diff
@@ -0,0 +1,201 @@
+var edx = edx || {},
+    host = window.location.host.substring(0, window.location.host.indexOf(':')),
```

Comment:
It was originally used to only return the second item if on localhost, but only the test uses that second item returned so I don't know if this is necessary.